advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx

```diff
@@ -0,0 +1,233 @@
+             |              |          |                            |                            |     | enterprisedb=CTc/enterprisedb
+(5 rows)
+
+
+edb=#
+edb=# \c epas13_test
+You are now connected to database "epas13_test" as user "enterprisedb".
+epas13_test=# \dt
+                List of relations
+ Schema |       Name       | Type  |    Owner
+--------+------------------+-------+--------------
+ public | tp_department_db | table | enterprisedb
+ public | tp_sales_db      | table | enterprisedb
+(2 rows)
+
+
+epas13_test=# select * from  tp_department_db;
+ deptno |    dname    | location
+--------+-------------+----------
+     10 | Development | Pakistan
+     20 | Testing     | Pakistan
+     30 | CM          | Pakistan
+     40 | Marketing   | India
+(4 rows)
+
+
+epas13_test=# select * from  tp_sales_db;
+ salesman_id | salesman_name | sales_region | sales_amount | deptno
+-------------+---------------+--------------+--------------+--------
+         100 | Person 1      | CITY 1       |            1 |     10
+         110 | Person 2      | CITY 2       |            2 |     20
+         120 | Person 3      | CITY 3       |            3 |     30
+         130 | Person 4      | CITY 4       |        10000 |     40
+(4 rows)
+
+
+epas13_test=# select * from v1;
+ dept_no |  dept_name  | sales_no | sales_name | sales_salary | sales_dept_no
+---------+-------------+----------+------------+--------------+---------------
+      10 | Development |      100 | Person 1   |            1 |            10
+      20 | Testing     |      110 | Person 2   |            2 |            20
+      30 | CM          |      120 | Person 3   |            3 |            30
+      40 | Marketing   |      130 | Person 4   |        10000 |            40
+(4 rows)
+
+
+epas13_test=# desc  tp_sales_db;
+                                 Table "public.tp_sales_db"
+    Column     |         Type          | Collation | Nullable |           Default
+---------------+-----------------------+-----------+----------+------------------------------
+ salesman_id   | integer               |           |          |
+ salesman_name | character varying(30) |           |          |
+ sales_region  | character varying(30) |           |          |
+ sales_amount  | integer               |           |          | nextval('sal_seq'::regclass)
+ deptno        | integer               |           |          |
+Indexes:
+    "lower_reg_idx" btree (lower(sales_region::text))
+    "reg1_idx" btree (salesman_id)
+Foreign-key constraints:
+    "department_employee_fk" FOREIGN KEY (deptno) REFERENCES tp_department_db(deptno)
+
+
+epas13_test=#
+epas13_test=#
+
+```
+
+### FSBasedBackupSet Backup and Restore
+
+#### Taking FSBasedBackupSet Backup 
+
+!!! Note
+    For FSBasedBackupSet Backup WAL Arching must be enabled on EDB Postgres Advanced Server (EPAS). Visit the link for details https://www.postgresql.org/docs/13/continuous-archiving.html.
+
```

Comment:
We can take out all the steps and just leave the text I proved above.  
You should make sure to keep this information in a word doc so that when it is fixed you have it to add back.